pkg/storaged/utils.js

```diff
@@ -418,7 +446,15 @@ export function get_available_spaces(client) {
         const block = client.blocks[path];
         const parts = get_block_link_parts(client, path);
         const text = cockpit.format(parts.format, parts.link);
-        return { type: 'block', block, size: block.Size, desc: text };
+        return {
+            type: 'block',
+            block,
+            size: block.Size,
+            desc: text,
+            type_label: parts.type,
+            name: parts.name,
+            subname: parts.subname
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)